src/macaron/slsa_analyzer/build_tool/gradle.py

```diff
@@ -135,3 +136,42 @@ def get_dep_analyzer(self, repo_path: str) -> CycloneDxGradle:
             )
 
         raise DependencyAnalyzerError(f"Unsupported SBOM generator for Gradle: {tool_name}.")
+
+    def get_group_id(self, project_path: str) -> str | None:
+        """Get the group id of a Java repository.
+
+        Parameters
+        ----------
+        project_path : str
+            Path to the repository.
+
+        Returns
+        -------
+        str | None
+            The group id, if exists.
+        """
+        # Use the gradlew that comes with the repository first, then use Macaron's
+        # built-in gradlew as a fallback option.
+        if os.path.isfile(os.path.join(project_path, "gradlew")):
+            gradlew = "./gradlew"
+        else:
+            gradlew = os.path.join(global_config.resources_path, "gradlew")
+        try:
+            result = subprocess.run(  # nosec B603
+                [gradlew, "properties"],
+                capture_output=True,
+                cwd=project_path,
+                check=False,
+            )
+        except (subprocess.CalledProcessError, OSError) as error:
+            logger.debug("Could not capture the group id of the repo at %s", project_path)
```

Comment:
@behnazh-w You asked in [this comment](https://github.com/oracle/macaron/pull/349#discussion_r1257570447)
> This can be a debug level log?

It is fixed now.